src/peft/utils/loftq_utils.py

```diff
@@ -0,0 +1,190 @@
+import torch
+from scipy.stats import norm
+
+
+class NFQuantizer:
+    def __init__(self, num_bits=2, device="cuda", method="normal", block_size=64, *args, **kwargs):
```

Comment:
Is there any way for the user to control what `method` is being used here? If not, should it be added as an option or should the `uniform` method just be removed?